requirements/base.in

```diff
@@ -0,0 +1,28 @@
+# Core requirements for using this application
+-c constraints.txt
+
+# edX Internal Requirements
+edx-i18n-tools
+edx-submissions>=2.0.12
+git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework>=3.6.3
+git+https://github.com/edx/XBlock.git@xblock-1.0.1#egg=XBlock>=1.0.1
+git+https://github.com/edx/xblock-sdk.git@331e50bed2a3c9014d998cc4428e92bf658baf8d#egg=xblock-sdk>=0.1.6
+
+django>=1.8,<2.0
+
+# Third Party Requirements
+bleach>=2.1.4
+html5lib>=1.0.1
+boto>=2.39.0,<3.0.0
```

Comment:
...and for anything that has an upper limit version constraint, move that into `constraints.txt`.  e.g. move `boto<3.0.0` into `constraints.txt` and remove it from this file.